proposals/lip-0061.md

```diff
@@ -570,7 +573,9 @@ def verifyAggregateCommit(blockHeader: BlockHeader)-> bool:
     # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated
     # by the previous certificate.
     heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1)
-    if previous function call returns valid height and aggregateCommit.height > heightNextBFTParameters-1:
+    if previous function call returns valid height
+     and heightNextBFTParameters > MIN_CERTIFICATE_HEIGHT
```

Comment:
So basically here we amend to the rule state in the comment as:

`If there are new BFT parameters for a height h > MIN_CERTIFICATE_HEIGHT, then the chain needs to include an aggregate commit for the block at height h-1.`

Reply:
Exactly.